.github/workflows/periodic.yaml

```diff
@@ -0,0 +1,50 @@
+# Workflow to create an auto-buildable release periodically.
+#
+# Releases created with automation credentials don't trigger workflows.
+# Therefore a token `secrets.RELEASE_TOKEN` must exist with public_repo scope.
+name: Periodic release
+on:
+  # Run weekly on sunday at 21:37 UTC (arbitrary)
```

Comment:
why not 13:37 :( ?

Reply:
Heh, habit of avoiding popular scheduling times like 00:00 or 13:37 ;-)